package.json

```diff
@@ -18,17 +18,25 @@
     "lint:js": "eslint .",
     "start": "ember serve",
     "test": "ember test",
-    "test:all": "ember try:each"
+    "test:all": "ember try:each",
+    "prepublishOnly": "ember ts:precompile",
+    "postpublish": "ember ts:clean"
   },
   "dependencies": {
     "ember-cli-babel": "^7.11.1",
```

Comment:
Please update this to latest (to prep for a future ember-cli-typescript@4 update).

Reply:
Done!